### ISSUE TITLE
"First setup" doc not found

### ISSUE THREAD

**Issue by isbkch** [opencall/opencall#26]
I've been looking for the "first setup" document.
The project seems dead but if someone happen to know when can I get...

**Comment by thoroc** [comment]
Hi,
OpenCall is basically a Symfony2 project. You should be able to install it like you would do for any SF2 project: once cloned, run composer to install the vendors and adjust the settings in `/app/config/`

**Comment by isbkch** [comment]
Yes, I managed to do that, It's more about the "how to connect it to receive phone calls" part

**Comment by thoroc** [comment]
Sorry for that you'll need to dive into the third party lib this project was intending to use: [Plivo](https://www.plivo.com/open-source/)

**Comment by isbkch** [comment]
Alright, thank you man, it's too bad what happend to the project..
Were you able to install/configure/run it ?

**Comment by rovico** [comment]
So sad, that project is dead :(
